resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -1109,6 +1111,37 @@ function resize() {
         @if ($gameModel->system->active && !$isEventGame)
             <x-game.leaderboards-listing :game="$gameModel" />
         @endif
+
+        @if (count($gameModel->event?->awards ?? []) > 0)
+            <?php
+                $badgeCounts = PlayerBadge::where('AwardType', AwardType::Event)
+                    ->where('AwardData', $gameModel->event->id)
+                    ->groupBy('AwardDataExtra')
+                    ->select(['AwardDataExtra', DB::raw('count(*) AS total')])
+                    ->get();
+            ?>
+            <div class="component gamealts">
+                <h2 class="text-h3">Award Tiers</h2>
+                <table class="table-highlight"><tbody>
+                @foreach ($gameModel->event->awards->sortBy('acheivements_required') as $award)
+                    <tr style="width:100%">
+                        <td style="width:100%">
```

Comment:
nit: If possible, I recommend using Tailwind classes for these inline styles.
```
<tr class="w-full">
    <td class="w-full">

...

<td class="text-right">
```

Reply:
done